### DISCUSSION TITLE
No Arch Linux release?

### DISCUSSION THREAD

**Discussion by Flameopathic** [goastian/midori-desktop#56] [General]
I may be missing something, but it seems as if the most recent version of Midori lacks a way to natively install on Arch Linux, unlike past releases. Is there a specific reason for this, or did I miss the download?

**Comment by Ponchale** [comment]
@Flameopathic Friend, you can check the package for ArchLinux is now available from the repository https://github.com/goastian/midori-desktop/releases/tag/v11.2.2
